helm/nginx-ingress-controller-app/templates/controller-deployment.yaml

```diff
@@ -72,11 +71,12 @@ spec:
           runAsUser: {{ .Values.initContainer.userID }}
           runAsGroup: {{ .Values.initContainer.groupID }}
       containers:
-      - name: {{ .Values.controller.name }}
+      - name: {{ include "name" . }}
         image: "{{ .Values.image.registry }}/{{ .Values.controller.image.repository }}:{{ .Values.controller.image.tag }}"
         args:
         - /nginx-ingress-controller
-        - --configmap=$(POD_NAMESPACE)/{{ .Values.controller.configmap.name }}
+        - --configmap={{ .Release.Namespace }}/{{ include "resource.default.name" . }}
+        - --election-id={{ include "resource.default.name" . }}-leader
```

Comment:
Got this from https://kubernetes.github.io/ingress-nginx/user-guide/multiple-ingress/